syntax = "proto3";

message TypeA {
	string field1 = 1;
	int32 field2 = 2;
}

message TypeB {
	int32 field1 = 1;
	int32 field2 = 2;
	int32 field3 = 3;
}
